engine/src/flutter/lib/web_ui/lib/src/engine/canvas_pool.dart

```diff
@@ -1244,6 +1255,13 @@ class _SaveStackTracking {
     clipStack!.add(SaveClipEntry.rrect(rrect, _currentTransform.clone()));
   }
 
+  /// Adds a round rectangle to clipping stack.
+  @mustCallSuper
+  void clipRSuperellipse(ui.RSuperellipse rse) {
+    clipStack ??= <SaveClipEntry>[];
+    clipStack!.add(SaveClipEntry.rrect(rse.toApproximateRRect(), _currentTransform.clone()));
```

Comment:
To simplify this and guarantee that it stays in sync with `clipRRect`, may I suggest:
```suggestion
    clipRRect(rse.toApproximateRRect());
```

Reply:
Done!